packages/app-desktop/gui/NoteListControls/NoteListControls.tsx

```diff
@@ -69,6 +71,73 @@ const SortOrderButtonsContainer = styled.div`
 
 function NoteListControls(props: Props) {
 	const searchBarRef = useRef(null);
+	const newNoteRef = useRef(null);
+	const newTodoRef = useRef(null);
+	const noteControlsRef = useRef(null);
+	const searchAndSortRef = useRef(null);
+
+	const [noteBtnText, setNoteBtnText] = useState('New note');
+	const [todoBtnText, setTodoBtnText] = useState('New to-do');
+	const [breakpoint, setBreakpoint] = useState('l');
```

Comment:
The l/m/xl/etc values should be defined as an enum (which you could put in the same place as the ValueMap interface

Reply:
I created an enum for the values. Depending on what we do with `ValueMap` I'll move them together.